testing/zerotier-one/zerotier-one.confd

```diff
@@ -0,0 +1 @@
+zerotier_opts=""
```

Comment:
Well, that’s true, but only when there are some extra variables not defined by OpenRC. However, I’m not strongly against adding this file, so okay.